eth/vm/forks/byzantium/__init__.py

```diff
@@ -70,10 +76,10 @@ class ByzantiumVM(SpuriousDragonVM):
     _state_class = ByzantiumState  # type: Type[BaseState]
 
     # Methods
-    create_header_from_parent = staticmethod(create_byzantium_header_from_parent)
-    compute_difficulty = staticmethod(compute_byzantium_difficulty)
+    create_header_from_parent = staticmethod(create_byzantium_header_from_parent)   # type: ignore
+    compute_difficulty = staticmethod(compute_byzantium_difficulty)     # type: ignore
```

Comment:
Can we use comment type hints to give these methods the proper `Callable` type instead of ignoring them or is there a reason why that wouldn't work?

Reply:
Turns out these are a bit harder to solve and I'm ok to address these in another PR